v1/random.yaml

```diff
@@ -37,7 +37,7 @@ paths:
               status: 303
               headers:
                 cache_control: '{{options.random_cache_control}}'
-                location: '../../{request.params.format}/{title_from_mobileapps.body.title}'
+                location: '../../{request.params.format}/{title_from_mobileapps.body.items[0].title}'
```

Comment:
Even though I too find in superfluous, let's keep it as-is now for sanity's sake. Also, it needs to stay as-is if we want to get it out tomorrow.